README.rst

```diff
@@ -368,14 +368,17 @@ pyproject.toml
 --------------
 
 autopep8 can also use ``pyproject.toml``.
-section must use ``[tool.autopep8]``, and ``pyproject.toml`` takes precedence
+The section must use ``[tool.autopep8]``, and ``pyproject.toml`` takes precedence
 over any other configuration files.
 
 configuration file example::
 
     [tool.autopep8]
     max_line_length = 120
     ignore = "E501,W6"  # or ["E501", "W6"]
+    in-place = true
+    recursive = true
+    aggressive = true
```

Comment:
Can you change `aggressive` to 1, 2, or 3?

Reply:
I tried that, it doesn't work. Specifying any integer works the same as true.

Reply:
It's working.

```
$ autopep8 --version
autopep8 1.5.6 (pycodestyle: 2.7.0)

$ autopep8 -d hello.py
--- original/hello.py
+++ fixed/hello.py
@@ -1 +1,2 @@
-def f(x): return 2 * x
+def f(x):
+    return 2 * x

$ vim pyproject.toml

$ cat pyproject.toml
[tool.autopep8]
ignore = ["W"]
diff = true
aggressive = 1
$ autopep8 -d hello.py

$
```

Reply:
Huh, indeed it is, no idea what I was doing then 😅 
Gitlab seems to have decided I'm not allowed to edit this patch any more, so I'll close this and make a new one